File: 프로그래머스/lv1/76501. 음양 더하기/음양 더하기.js
function solution(absolutes, signs) {
    
  let result = 0
  
  for(let i = 0; i < absolutes.length; i++){
    for(let j = 0; j < signs.length; j++){
      if( i === j && signs[j] === true){
          result = result + absolutes[i]
         
      }
      else if( i === j && signs[j] === false){
        result = result - absolutes[j]
      }
     
    }
    
  }
  return result
}

File: 프로그래머스/lv1/76501. 음양 더하기/음양 더하기.py
# 
def solution(absolutes, signs):
    answer = 123456789
    for i in range(len(absolutes)):
        #print(absolutes[i], signs[i])
        if signs[i] == False:
            absolutes[i] *= -1
        else:
            continue
    #print(absolutes)
    return sum(absolutes)
  